app/views/posts/_list_item.haml

```diff
@@ -1,5 +1,6 @@
 %tr
-  - klass = cycle('even', 'odd')
+  - klass = [cycle('even', 'odd')]
```

Comment:
I don't think this works since we're inserting the array into another array below; did it work when tested? We might need a .flatten somewhere.